docs/source/starters/index.md

```diff
@@ -0,0 +1,44 @@
+# New project tools
+As you saw from the [First steps](../get_started/new_project.md) section, once you have [set up Kedro](../get_started/install.md), you can use `kedro new` to create a basic Kedro project containing project directories and basic code, which you can configure depending on the tooling and example code you need.
+
+There are several ways to create a new Kedro project, which the pages in this section describe in detail.
```

Comment:
Isn't there really just one way? `kedro new` and `kedro new` with project tools are variations of a single way, right? Or at least, from the user perspective I don't think the distinction is that important.

One could consider the default `kedro new` as a "default starter", even if it's not technically that.

As far as I understand, the project tools apply to all starters, right?

Reply:
Correct, `--starter` and `--tools` cannot be used together, neither can `--starter` and `--example`.

Reply:
Oh I was massively confused. Thanks